common/persistence/persistence-tests/executionManagerTest.go

```diff
@@ -1671,7 +1671,7 @@ func (s *ExecutionManagerSuite) TestDeleteWorkflow() {
 // TestDeleteCurrentWorkflow test
 func (s *ExecutionManagerSuite) TestDeleteCurrentWorkflow() {
 	if s.ExecutionManager.GetName() != "cassandra" {
-		s.T().Skip("this test is only applicable for cassandra (uses TTL based deletes)")
+		return
```

Comment:
I think skip is the correct behavior here. Does it cause any issue for you?

Reply:
Yes it causes issue that the whole TestSQLExecutionManagerSuite will skip, when I run postgres test locally.
<img width="1083" alt="Screen Shot 2020-09-10 at 2 43 51 PM" src="https://user-images.githubusercontent.com/4523955/92812509-4655b780-f374-11ea-84da-047d56701e93.png">

Anyway it's already committed in a previous PR when I fixed the Postgres upsert issue.